### PR TITLE
RHDS terminology

### PR DESCRIPTION
**Pull request by mportman12**
Add RHDS terminology from this doc: https://docs.google.com/document/d/1uyD9i8Vbfa75KbGDPrGdxYIa8UFlH4PNPbD7Eltp4Gk/edit

### CODE REVIEW

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (inoxx03) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:8]:
```suggestion
*Description*: In an LDAP directory, an access control instruction (ACI) grants or denies access to entries or attributes. Use access control instruction on the first usage and then the abbreviation ACI subsequently.
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:50]: When I did a build, I could see that this ID was already in use. Looks to be in the AMQ section. Might just need to make it "consumer-rhds" or something like that

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:8]: Suggest putting "access control instruction" and "ACI" in quotes in the second sentence.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:30]: By chance is there any guidance on when to write out "base distinguished name (DN)" and when they can just say "base DN"? Because the entry is "base DN" but the description writes it out fully.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:74]: Suggest putting "Red Hat Directory Server" and "Directory Server" in quotes in the second sentence (places where you tell them which words to use).

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:151]: Just curious, if you were writing about this objectClass, would you mark it up in monospace as `objectClass`? If so, I would do that here too. (And if not, then ignore this! :) )

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:184]: slave -> "slave"

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:195]: TLS -> "TLS" in last sentence

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:245]: Since SSL links here, should TLS link back to it as well?

**Review by bergerhoffer — COMMENTED** [review]
Thanks @mportman12 for submitting this! Just a few comments, mostly just on adding quotes where we say to use a particular term

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:30]: This gets explained in the DN entry below - "distinguished name" on first usage, then "DN" subsequently.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:151]: I'm not the RHDS writer, so not sure! I can ask Marc if you'd prefer me to follow up on this.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed and updated xrefs to match.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:193]: Looks like there is another "ssl" ID already too sorry, can you also fix this to be ssl-rhds or something else unique?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:151]: Nah, not a big deal, thanks

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

**Review by bergerhoffer — DISMISSED** [review]
Thanks for the updates, this LGTM now!

@redhat-documentation/ccs-style-council this is ready for more reviews please!

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bburt-rh) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:63]:
```suggestion
*Description*: The privileged administrative user in Red Hat Directory Server is called the Directory Manager. The distinguished name (DN) of this user is cn=Directory Manager.
```
Not sure about you, but adding the direct article here sounds more idiomatic to my ears. What do you think?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (bburt-rh) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc:74]:
```suggestion
*Description*: The short product name of Red Hat Directory Server. In the title of guides, use the full product name "Red Hat Directory Server" and, elsewhere, the short name "Directory Server". Because it is the product name, both words start with a capital letter. Additionally, this practice distinguishes the Red Hat Directory Server product from other directory servers.
```
Consider adding a pronoun reference. It's ambiguous what "this" refers to.

**Comment by bburt-rh** [comment]
Other than a couple of nitpicks, LGTM. Thanks!

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc]: Fixed

**Review by inoxx03 — DISMISSED** [review]

**Review by bergerhoffer — APPROVED** [review]
@mportman12 This LGTM still, but it has a conflict if you can resolve that please

**Review by bergerhoffer — APPROVED** [review]

**Comment by mportman12** [comment]
@bergerhoffer Can I merge this PR?

**Comment by bergerhoffer** [comment]
Agree, it had previous approvals before the merge conflict. Merging!

**Pull request merged**
